frontend/test/playwright/e2e/seo.spec.ts

```diff
@@ -8,6 +8,7 @@ const DESCRIPTION =
 const NO_INDEX = "noindex, nofollow"
 const INDEX =
   "index, follow, max-image-preview:large, max-snippet:-1, max-video-preview:-1"
+const INDEX_NO_FOLLOW = "index, nofollow"
```

Comment:
I wonder if this should have the `max-` directives as well, if index is enabled?